src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1219,13 +1252,16 @@ public void UpdateBubbleContent()
             foreach (var info in NodeModel.Infos)
             {
                 var infoStyle = info.State == ElementState.Error ? InfoBubbleViewModel.Style.Error : InfoBubbleViewModel.Style.Warning;
+                infoStyle = info.State == ElementState.Info ? InfoBubbleViewModel.Style.Info : infoStyle;
                 var data = new InfoBubbleDataPacket(infoStyle, topLeft, botRight, info.Message, connectingDirection);
                 packets.Add(data);
             }
             InfoBubbleViewModel.Style style = NodeModel.State == ElementState.Error
                 ? InfoBubbleViewModel.Style.Error
                 : InfoBubbleViewModel.Style.Warning;
 
+            style = NodeModel.State == ElementState.Info ? InfoBubbleViewModel.Style.Info : style;
```

Comment:
Same here, the info state cannot override the error state. I think the hierarchy of states should be:
1. Error 2. Warning 3. Info
@Amoursol please confirm.

Reply:
Correct @aparajit-pratap, the hierarchy of the graphical representation is as follows when **collapsed:** Error > Warning > Info

Beyond that, they should all exist there in a stack when **uncollapsed,** so when you hover over that hierarchy bases graphical representation you still see anything else that exists. Refer to the Figma here: https://www.figma.com/file/UIDVMHnpK4TL6gm1U7QKEh/Dynamo_Redesign%3A-Alerts?node-id=0%3A1